### PR TITLE
Added CSS

### PR DESCRIPTION
**Pull request by Quelklef**
CSS!!

### CODE REVIEW

**Comment by samm81** [comment]
It doesn't appear to look any different to me. What are you trying to accomplish? Perhaps some before/after pictures would help?

File: static/css.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#meal-selection .jumbotron,
+#location-selection .jumbotron {
+    cursor: pointer;
+    position: relative;
+    box-shadow: 0 0 16px rgba(0, 0, 0, 0.1);
+    transition: box-shadow 0.5s;
+}
```
Comment (samm81) [inline static/css.css]: newlines after closing braces please

File: templates/layout.html [hunk 2]
```diff
@@ -17,6 +17,7 @@
 		<script type="text/javascript" src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
 		<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
 
+		<link href="{{ url_for('static', filename='css.css') }}">
```
Comment (samm81) [inline templates/layout.html]: use a better filename, like main.css

**Comment by Quelklef** [comment]
It's not hugely different. The main change can be found when selecting breakfast/lunch/dinner and the location.

**Comment by Quelklef** [comment]
Come to think of it, I never checked if the CSS file directly links... BRB.

**Pull request closed without merging**